src/ucp/core/ucp_context.c

```diff
@@ -283,9 +290,15 @@ static ucs_config_field_t ucp_config_table[] = {
    "and the resulting performance.\n",
    ucs_offsetof(ucp_config_t, ctx.estimated_num_ppn), UCS_CONFIG_TYPE_ULUNITS},
 
-  {"RNDV_FRAG_SIZE", "512k",
-   "RNDV fragment size \n",
-   ucs_offsetof(ucp_config_t, ctx.rndv_frag_size), UCS_CONFIG_TYPE_MEMUNITS},
+  UCP_CONFIG_RNDV_FRAG_SIZE(HOST, "512k"),
+  UCP_CONFIG_RNDV_FRAG_SIZE(CUDA, "4M"),
```

Comment:
maybe define it like
RNDV_FRAG_SIZE="512k,cuda:4m" - which means "default is 512k, but cuda memory is 4m"
so could easily extend to other mem types